### DISCUSSION TITLE
[Admin UI] How to prevent selecting duplicate items when using the `Relation` field?

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#1757] [Q&A] [answered]
Hey
For example, I have a `posts` and a `videos` collection. The `posts` collection has a `Relation` field (to select video/videos) and I marked it as `Unique`. I don't want any of the posts to have duplicate videos in them. The `Unique` option works only if only 1 video is selected from both posts.

## What I'm looking for:

```
post1 = ['video1']
post2 = ['video2']
// cool
```

```
post1 = ['video1']
post2 = ['video1', 'video2']
// not cool
```

```
post1 = ['video1', 'video2']
post2 = ['video1', 'video2']
// not cool
```

**Comment by ganigeorgiev** [comment, marked as answer]
Unique constraint compares only the value as a whole, eg. `video1` vs `video1,video2`. I'm planning to add Unique options in the next release to allow customizing the constraint behavior and will consider your use case.

Depending on the desired workflow, one workaround could be to remove your multi-`relation` field and fallback to a more traditional SQL structure using a junction collection (eg. `postsVideos`) that will have only 2 **single** `relation` fields pointing to your `posts` and `videos` collections. The main drawback is that you'll have to send a second request when create/update a `posts` but this will allow you to mark the `videos` relation field "Unique" and disallowing users to submit the same `videos` value.

**Reply by babakfp** [reply]
Cool
So I don't know how to do that, so I'm not going to😅 Thanks
